thunderfish/eodanalysis.py

```diff
@@ -16,32 +17,28 @@ def eod_extracting(bwin_data, samplerate, fish_type, psd_type):
                                 wave or pulsetype.
     :param psd_type:            (string) result of the psdtype algorithm that assigns the powerspectrum to wave or
                                 pulsetype.
+    :param pulse_window:        (float) data window around the detected EODs that shall be extracted in sec.
     :return eod_data:           (2-D array) An array of lists. Each list contains the data around a EOD.
     """
-    # ToDo: replace this with the peakdetection modul as soon as it is trustworthy.
     th = np.mean(bwin_data) + ( (np.max(bwin_data)-np.mean(bwin_data)) / 2.0 )
```

Comment:
Yes, we should use the percentile base threshold Juan is now using.
We should put this threshold estimation as an extra function into the peakdetection modul.